samples/Templates/Scenarios/CarouselWhenShowCarousel.template.json

```diff
@@ -0,0 +1,41 @@
+{
+	"type": "AdaptiveCard",
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"version": "1.6",
+	"body": [
+		{
+			"type": "Container",
```

Comment:
nit: i think it would be nicer to remove this top level Container. The inclusion of the Container in the past was it was only way to include both Carousel and Container card. Since the updated, users don't need this top level container.